examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -600,7 +622,22 @@ def compute_metrics(eval_preds):
                 recipes = {"smooth_quant": True, "smooth_quant_args": {"alpha": optim_args.smooth_quant_alpha}}
             else:
                 recipes = {}
-            quantization_config = PostTrainingQuantConfig(approach=optim_args.quantization_approach, recipes=recipes)
+            if optim_args.quantization_approach == "weight_only":
+                op_type_dict = {

```

Comment:
What happens if `quantization_approach == "weight_only"` and  `op_type_dict`  is not provided ? Should this be also added before quantization in the quantizer when not provided ?

Reply:
Also could make sense to keep the same API as for the NNCF quantizer : https://github.com/huggingface/optimum-intel/blob/v1.10.1/optimum/intel/openvino/quantization.py#L115

Reply:
if op_type_dict is not provided, it will use the default configuration: dtype-int4, group_size-32, scheme-sym, algorithm-RTN